packages/protocol/package.json

```diff
@@ -18,8 +18,7 @@
   "main": "index.js",
```

Comment:
remove the `index.js` in root and just reference the one from dist here, also add `"types": "dist/src/index.d.ts"` (or whatever the proper path is)